tests/src/integration/tests/test_prepare_on.cpp

```diff
@@ -369,7 +369,7 @@ CASSANDRA_INTEGRATION_TEST_F(PrepareOnUpAndAddTests, NotPreparedOnUpWhenDisabled
 
   // Wait for the node to become available and verify no statements have been
   // prepared
-  wait_for_node_on_session(session, 1);
+  wait_for_node_on_session(session_for_node(1), 1);
```

Comment:
Minor test change.  wait_for_node_on_session() is querying system.local so to get a robust comparison we should always use a clean session, especially after bootstrapping a new node.  I was seeing these tests fail due to a belief that the node was unavailable; after digging in it looked as if the session was simply querying the wrong node.